src/plugin.js

```diff
@@ -364,8 +364,7 @@ const vr = function(options) {
       camera.updateProjectionMatrix();
     }
 
-    window.addEventListener('resize', onWindowResize, false);
-    window.addEventListener('vrdisplaypresentchange', onWindowResize, true);
+    player.on(['resize', 'fullscreenchange'], onWindowResize);
```

Comment:
the `resize` event means that the video has changed resolution. We don't currently have an event for the player changed size other than `fullscreenchange`, I don't think.

We probably still want to listen to `resize` and `vrdisplaypresentchange` on `window`. At least the `vrdisplaypresentchange`.